demo/site/src/webpack/gql-hash-loader.ts

```diff
@@ -35,7 +35,7 @@ const gqlHashLoader = function (this: LoaderContext<unknown>, source: string) {
         const fragmentVariables = Array.from(query.matchAll(/\${.*?}/g) ?? [])
             .map((m) => m[0])
             .join(" ");
-        if (query.match(/^\s*fragment\s+(\w+)\s+on\s+\w+/m)) {
+        if (query.match(/^\s*fragment\s+(\w+)\s+on\s+\w+/)) {
```

Comment:
Unrelated fix that makes the NewsListBlock query work, which includes a query PLUS a fragment